javaagent-extension-api/src/main/java/io/opentelemetry/javaagent/extension/internal/DeclarativeConfigPropertiesBridge.java

```diff
@@ -45,71 +45,88 @@
  *       common:
  *         string_key: value
  * </pre>
+ *
+ * <p>This class is internal and is hence not for public use. Its APIs are unstable and can change
+ * at any time.
```

Comment:
```suggestion
```

Reply:
This is a required comment for internal classes

Reply:
this class is not public